e2e/cypress/common/permissions/dashboard.ts

```diff
@@ -0,0 +1,24 @@
+import { pageIsPermitted, ProjectInfo } from './shared';
+
+function tryClickIfClickable(item: DataCy.Value) {
+  cy.gcy(item).then(($el) => {
```

Comment:
This only tests, you used correct `clickable` classes. But it doesn't test the case there is no clickable class and the user still has permissions to do that by mistake. In that case user get's error and this case doesn't cover that. 

You also know what should be and shouldn't be clickable before the test, so you don't need the condition, you can provide the expected values without conditional testing. 

The naming is also obscure. It doesn't only click if clickable, but it checks whether there is no error thrown when user clicks on clickable item.